SourceX/miniwin/ddraw.cpp

```diff
@@ -7,12 +7,16 @@
 
 namespace dvl {
 
+#ifdef USE_SDL1
+SDL_Surface *window;
+#else
 SDL_Window *window;
 SDL_Renderer *renderer;
```

Comment:
you could implement `SDL_Renderer` as `void` and give it a clear todo comment in sdl2_to_1_2_backports.h, that way it's gathered in one place and easy to find what is needs to be worked for SDL1.

Reply:
Done, although I'm concerned that now the codebase might build with SDL1 but not run.

Reply:
Should be fine as long as you don't assign a pointer to it 😅